data_steward/gcloud/bq/__init__.py

```diff
@@ -184,3 +183,86 @@ def define_dataset(self, dataset_id: str, description: str,
         dataset.location = "US"
 
         return dataset
+
+    def copy_datasets(self, input_dataset: str, output_dataset: str):
+        """
+        Copies tables from source dataset to a destination datasets
+
+        :param input_dataset: name of the input dataset
+        :param output_dataset: name of the output dataset
+        :return:
+        """
+        # Copy input dataset tables to backup and staging datasets
+        tables = super(BigQueryClient, self).list_tables(input_dataset)
+        for table in tables:
+            staging_table = f'{output_dataset}.{table.table_id}'
```

Comment:
nit: I think this may need to be fully qualified
```suggestion
            staging_table = f'{self.project}.{output_dataset}.{table.table_id}'
```